### PR TITLE
Overwrite default relayer config with configs used to setup chains

### PR DESCRIPTION
**Pull request by MaCong-crypto**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -197,24 +197,16 @@ ibc-1:
         params:
           receive_enabled: true
           send_enabled: true
-relayer:
+relayer: # refer here (https://hermes.informal.systems/config.html) for more configs
   global:
-    timeout: 10s
-    light-cache-size: 20
-  paths:
```
Comment (yihuang) [inline README.md:204]: paths seem not used anywhere, I guess it's not needed for hermes? Have you tested it already?

Comment (MaCong-crypto) [reply README.md:204]: Yes, I have tested, `paths` is not used and needed for hermes.
Only configurations [here](https://hermes.informal.systems/config.html) are used by Hermes.

**Review by yihuang — APPROVED** [review]
LGTM

**Comment by yihuang** [comment]
Could you also add a changelog?

**Comment by yihuang** [comment]
It seems your first commit is not signed?

**Comment by MaCong-crypto** [comment]
> It seems your first commit is not signed?

Sorry, just signed it

**Pull request merged**